lib/offsite_payments/integrations/world_pay.rb

```diff
@@ -230,6 +230,12 @@ def custom_params
           return @custom_params ||= read_custom_params
         end
 
+        # Check if the request comes from IP range 195.35.90.0 – 195.35.91.255
+        def valid_sender?(ip)
+          return true if OffsitePayments.mode == :test 
+          !!(/^195\.35\.(9[0-1])\.([0-9]|[1-9][0-9]|1([0-9][0-9])|2([0-4][0-9]|5[0-5]))$/.match(ip))
```

Comment:
This is a hairy regex. Could you instead use `IPAddr#include?`?: http://ruby-doc.org/stdlib-2.0/libdoc/ipaddr/rdoc/IPAddr.html#method-i-include-3F